docs/user_guide/examples.md

```
@@ -91,6 +91,57 @@ lastly, running SIPL accelerated network python example on AGX Thor:
+
+1. **Resolution Changes**: Switching resolutions (not FPS) requires a camera reboot. The first resolution set after reboot will be used.
+
+2. **IP Address Discovery**: The camera IP may not be the default `192.168.0.2`. To discover the actual IP address, run the `hololink-enumerate` tool from within the Docker container:
+
+```sh
+$ hololink-enumerate
+```
+
+The output should display information similar to:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)